site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,461 @@
+---
+id: silabs-openthread-hardware
+summary: In this Codelab, you'll program OpenThread on real hardware, create and manage a Thread network, and pass messages between nodes.
+status: [draft]
+authors: Mithil Raut
+categories:
+tags:
+feedback link: https://github.com/openthread/ot-docs/issues
+
+---
+
+# Build a Thread network with Silicon Labs EFR32 boards and OpenThread using Simplicity Studio v5
+
+[Codelab Feedback](https://github.com/openthread/openthread/issues)
+
+## Introduction
+
+Duration: 3:00
+
+
+
+<img src="img/26b7f4f6b3ea0700.png" alt="26b7f4f6b3ea0700.png" width="624.00" />
+
+Google's [OpenThread](https://openthread.io) (OT) is an open-source implementation of Thread. Google has released OpenThread to make the networking technology used in Google Nest products more broadly available to developers, in order to accelerate the development of products for the connected home and commercial buildings. With a narrow platform abstraction layer and a small memory footprint, OpenThread is highly portable. It supports both system-on-chip (SoC) and network co-processor (NCP) designs.
+
+The  [Thread specification](http://threadgroup.org/ThreadSpec) defines an IPv6-based reliable, secure, and low-power wireless device-to-device communication protocol for home and commercial building applications.
+
+[Silicon Labs](https://www.silabs.com/) has enhanced OpenThread to work with Silicon Labs hardware. This source code is available on GitHub and also as a software development kit (SDK) installed with Simplicity Studio 5 (SSv5). The SDK includes a fully tested snapshot of the GitHub source code. It supports a broader range of hardware than does the GitHub version, and includes documentation and example applications not available on GitHub.
+
+This guide describes how to get started developing OpenThread applications using the Silicon Labs OpenThread SDK and Simplicity Studio 5. The image below shows the boards (BRDs) and the hardware set up with an OT Border Router (OTBR) and two Thread devices used in the codelab.
+
+> You can adapt this codelab to use any of the boards listed under hardware in the prerequisite section.
+
+![EFR32MG Hardware set up](./img/EFR32MG_hardware_setup.jpg)
+
+### What you'll learn
+
+* How to create an OpenThread project using Silicon Labs Simplicity Studio IDE.
+* How to build and flash OpenThread CLI binaries to Silicon Labs radio boards.
+* How to set up a Raspberry Pi 3B+ or greater as an OpenThread Border Router (OTBR) using Docker.
+* How to create a Thread network on the OTBR.
+* Out of band commissioning of devices onto a Thread network.
+* How to verify Thread communication between the nodes using the ping command.
+
+## Prerequisites
+
+Duration: 3:00
+
+Hardware:
+
+1. 3 EFR32MGxx radio boards - you can use any combination of these devices. I used a BRD4166A as the RCP and two BRD4168As as Full Thread Devices.
```

Comment:
Don't refer to yourself in the text, use something like "this codelab uses a BRD4166A..."